app/src/main/index.js

```diff
@@ -180,19 +192,37 @@ async function startTendermint (root) {
     'node',
     '--home', root
   ]
-  if (DEV) args.push('--log_level', 'info')
+  // if (DEV) args.push('--log_level', 'info')
```

Comment:
Why is this commented out? Won't this mean development mode now uses `error` log level? We might miss some useful messages.

Reply:
Was recommended by Frey

Reply:
OK